int64 tot_people
person_roi[] person